src/subdomains/supporting/support-issue/services/support-issue.service.ts

```diff
@@ -121,6 +115,38 @@ export class SupportIssueService {
     await this.messageRepo.save(entity);
 
     if (dto.author !== CustomerAuthor) await this.supportIssueNotificationService.newSupportMessage(entity);
+
+    return SupportIssueDtoMapper.mapSupportMessage(entity);
+  }
+
+  async getSupportIssue(userDataId: number, query: GetSupportIssueFilter): Promise<SupportIssueDto> {
+    if (!query.type && !query.id) throw new BadRequestException('Type or id is required');
```

Comment:
Use can use a `@ValidateIf` to enforce that one of the two fields must be set (see e.g. `src/subdomains/generic/user/models/user/dto/ref-info-query.dto.ts`)